x-pack/plugins/ml/server/models/job_validation/validate_job_object.js

```diff
@@ -5,31 +5,56 @@
  */
 
 
+import { i18n } from '@kbn/i18n';
 
 export function validateJobObject(job) {
   if (job === null || typeof job !== 'object') {
-    throw new Error('Invalid job: Needs to be an object.');
+    throw new Error(i18n.translate('xpack.ml.models.jobValidation.jobIsNotObjectErrorMessage', {
+      defaultMessage: 'Invalid {invalidParamName}: Needs to be an object.',
+      values: { invalidParamName: 'job' },
+    }));
   }
   if (job.analysis_config === null || typeof job.analysis_config !== 'object') {
-    throw new Error('Invalid analysis_config: Needs to be an object.');
+    throw new Error(i18n.translate('xpack.ml.models.jobValidation.analysisConfigIsNotObjectErrorMessage', {
+      defaultMessage: 'Invalid {invalidParamName}: Needs to be an object.',
+      values: { invalidParamName: 'job.analysis_config' },
```

Comment:
```suggestion
      values: { invalidParamName: 'analysis_config' },
```

Reply:
I believe it was a typo in the source code because in all other places it is writted `'job.something'`. cc @peteharverson

Reply:
Looks like the text for the name of the fields is not consistent in that file. I would use the full path for all messages - so use `job.analysis_config` here